### ISSUE TITLE
Unable to reproduce the previously mentioned accuracy on NCI1, NCI109, Mutagenicity

### ISSUE THREAD

**Issue by wangyuhu** [psp3dcg/GSAPool#2]
Hello, I can’t reproduce the accuracy mentioned before on NCI1, NCI109, Mutagenicity； Could you tell me the hyperparameter settings on NCI1, NCI109, Mutagenicity? Thx!

**Comment by psp3dcg** [comment]
Hello, the alpha(hyper-parameter) on NCI-1, NCI-109 and Mutagenicity are all 0.4, hoping that can be helpful

**Comment by wangyuhu** [comment]
Ok, I'll try! Thanks very much!
